src/NLog/Targets/DefaultJsonSerializer.cs

```diff
@@ -692,14 +691,19 @@ internal static void AppendStringEscape(StringBuilder destination, string text,
                 destination.Append(text);   // Faster to make single Append
         }
 
-        internal static bool RequiresJsonEscape(char ch, bool escapeUnicode)
+        internal static bool RequiresJsonEscape(char ch, JsonSerializeOptions options)
+        {
+            return RequiresJsonEscape(ch, options.EscapeUnicode, options.EscapeForwardSlash);
+        }
+
+        internal static bool RequiresJsonEscape(char ch, bool escapeUnicode, bool escapeForwardSlash)
         {
             if (!EscapeChar(ch, escapeUnicode))
             {
                 switch (ch)
                 {
                     case '"':
-                    case '\\':
+                    case '\\': return escapeForwardSlash;
```

Comment:
Looks like you chose the backward-slash instead of the forward-slash?

Reply:
Maybe something like this:
```c#
               switch (ch)
                {
                    case '/': return escapeForwardSlash;
                    case '"':
                    case '\\':
                        return true;
```

Reply:
oops, thanks!